bot.py

```diff
@@ -128,20 +133,19 @@ async def exclusion(before, after):
 
         await modlog.send(embed = e)
 
+async def assign_base_roles(newMember, guild):
+    roles = [guild.get_role(x) for x in (708313061764890694, 708315631774335008, 754029717211971705, 708313617686069269, 856620435164495902, 596511307209900053, 717132666721402949, 1101606908437221436)]
```

Comment:
From now on we could use better coding practices, such as naming constants (in this case, have one constant per role to see which role id corresponds to which role).